modules/venat-module-market/src/dto/market.dto.ts

```diff
@@ -0,0 +1,29 @@
+import { Param } from '@discord-nestjs/core';
+import { Transform } from 'class-transformer';
+
+interface TextParameter {
+  value: string;
+}
+
+function cleanText({ value }: TextParameter): string {
+  const duplicateWhitespace = /(\s\s+)*/gm;
+  return value.normalize().replaceAll(duplicateWhitespace, '').trim();
+}
```

Comment:
Similar comment about library stuff - do we want to split utility functions like this out into a common library, or even just put them into the core directly? I can imagine this will get used in more than just this module.

Reply:
That's a good idea, I'll move these into the core for now.

Reply:
Updated